mezzanine/project_template/deploy/local_settings.py.template

```diff
@@ -15,7 +15,7 @@ DATABASES = {
         # Not used with sqlite3.
         "PASSWORD": "%(db_pass)s",
         # Set to empty string for localhost. Not used with sqlite3.
-        "HOST": "127.0.0.1",
+        "HOST": "localhost",
```

Comment:
Why this change? It's not guaranteed localhost exists (will probably never happen, but there's at least one reason for having it one way over the other).